gcc.spec

```diff
@@ -247,7 +252,7 @@ Man and info pages for %{name}.
 %package -n libgcc
 Summary: GCC version 10.3 shared support library
 Obsoletes: libgcc < %{version}-%{release}
-Autoreq: true
+Autoreq: false
```

Comment:
I guess this works as it translates to `0`, although https://rpm-software-management.github.io/rpm/manual/spec.html says 0/1, yes/no

```
AutoReqProv
AutoReq
AutoProv
Control per-package automatic dependency generation for provides and requires. Accepted values are 1/0 or yes/no, default is always “yes”. Autoreqprov is equal to specifying Autoreq and Autoprov separately.
```

Reply:
False seems to be valid value, Fedora spec also uses false in AutoReq. Tested the change and the Requires does change as expected.